examples_server/examples/data_table/basic/data_table.js

```diff
@@ -51,10 +51,9 @@ Backdraft.app("TableExample", function(app) {
   app.view.dataTable("BookTable", {
 
     rowClassName : "BookRow",
-
     paginate : false,
-
-    reorderableColumns: true
+    reorderableColumns: true,
+    // striped: false
```

Comment:
yeah I left it as intentional to show the potential option (it's in the examples_server, not in our library code..)